properties/src/main/java/gov/cms/ab2d/properties/service/PropertiesAPIServiceImpl.java

```diff
@@ -54,6 +118,15 @@ public boolean deleteProperty(String key) {
         if (StringUtils.isEmpty(key)) {
             return false;
         }
+        if (usePropertyService) {
+            try {
+                propertiesClient.deleteProperty(key);
+                return true;
+            } catch (Exception ex) {
+                log.error(ERROR_MESSAGE, ex);
+                return propertiesService.deleteProperty(key);
```

Comment:
Might be better to have an event call instead. Especially if in not too long we are deleting propertiesService soon.

Reply:
This is temporary because this is the fallback to the local database if the service isn't working.